src/commands/cni/test/flow.ts

```diff
@@ -116,20 +139,217 @@ export default class CniTestFlowCommand extends PrismaticBaseCommand {
+          fs.appendFile(resultFilePath, `${log.timestamp},${log.severity},${log.message}\n`);
+        }
+      }
+
+      if (await this.shouldEnd(executionId, autoEndPoll)) {
+        return;
+      }
+    }
+  }
+
+  private async tailStepResults(executionId: string, startTime: number) {
+    const {
+      flags: { "auto-end": autoEndPoll, "result-file": resultFilePath },
+    } = await this.parse(CniTestFlowCommand);
+
+    let nextCursor: string | undefined = undefined;
+
+    while (true) {
+      await ux.wait(this.getPollIntervalMs());
+
+      const result: any = await this.fetchStepResults(executionId, nextCursor);
+      if (result === undefined) {
+        continue;
+      }
+
+      const { stepResults, cursor } = result;
+      nextCursor = cursor;
+
+      ux.table(
+        stepResults,
+        {
+          endedAt: {},
+          type: {
+            minWidth: 15,
+          },
+          result: {},
+        },
+        {
+          "no-header": true,
+        },
+      );
+
+      if (resultFilePath) {
+        for (const result of stepResults) {
+          fs.appendFile(
+            resultFilePath,
+            `${result.endedAt},${result.type},${JSON.stringify(result.result)}\n`,
+          );
+        }
+      }
+
+      if (await this.shouldEnd(executionId, autoEndPoll)) {
+        return;
+      }
+    }
+  }
+
+  private async fetchLogs(
+    executionId: string,
+    nextCursor?: string,
+  ): Promise<FetchLogsResult | undefined> {
+    const results = await listExecutionLogs(executionId, nextCursor);
+
+    const { edges }: { edges: { node: LogNode; cursor?: string }[] } = results.logs;
+    if (!edges || edges.length === 0) {
+      return undefined;
+    }
+
+    const logs = edges.map(({ node }) => node);
+
+    const { cursor } = edges[edges.length - 1];
+    return { logs, cursor };
+  }
+
+  private async fetchStepResults(executionId: string, nextCursor?: string) {
+    const results = await listExecutionStepResults(executionId, nextCursor);
+
+    const { edges }: { edges: { node: StepResultNode; cursor?: string }[] } =
+      results.executionResult.stepResults;
+
+    if (!edges || edges.length === 0) {
+      return undefined;
+    }
+
+    const stepResults: Array<FormattedStepResult> = [];
+
+    for (const edge of edges) {
+      const { endedAt, resultsUrl, stepName } = edge.node;
+
+      const response = await axios.get(resultsUrl, {
+        responseType: "arraybuffer",
+      });
+      const resultsBuffer = Buffer.from(await response.data);
+      const result = decode(resultsBuffer) as Record<string, unknown>;
+
+      stepResults.push({
+        type: stepName === "onTrigger" ? "RESULT_TRIGGER" : "RESULT_FLOW",
+        endedAt,
+        result,
+      });
+    }
+
+    const { cursor } = edges[edges.length - 1];
+    return { stepResults, cursor };
+  }
+
+  private async executionIsComplete(executionId: string) {
+    return await cniExecutionIsComplete(executionId);
+  }
```

Comment:
```suggestion
  private async executionIsComplete(executionId: string) {
    return await cniExecutionIsComplete(executionId);
  }
```

This wrapper doesn't do anything special, how come you just do use `await cniExecutionIsComplete(executionId)` where you need it?